app/config.py

```diff
@@ -1 +1,7 @@
+import os
+
+
 DEBUG = True
+
+SQLALCHEMY_DATABASE_URI = os.getenv('DEED_DATABASE_URI',
```

Comment:
I guess this should be VERIFUDGE_DATABASE_URI (not DEED). It should be set in puppet/templates/service.systemd.erb